src/ray/util/event.cc

```diff
@@ -103,10 +103,43 @@ std::string LogEventReporter::EventToString(const rpc::Event &event,
   return j.dump();
 }
 
+std::string LogEventReporter::ExportEventToString(const rpc::Event &event,
+                                                  const json &custom_fields) {
+  json j;
+
+  auto timestamp = event.timestamp();
+
+  j["timestamp"] = timestamp;
+  j["event_id"] = event.event_id();
+  j["source_type"] = Event_SourceType_Name(event.source_type());
+  j["host_name"] = event.source_hostname();
+  j["pid"] = std::to_string(event.source_pid());
```

Comment:
Do we need all these?

Seems the minimal is `{"type": xxx, "data": yyyy}` where "type" is just telling us how to parse data and what schema is expected?

Reply:
We should keep `timestamp` because it's not included in the `event_data`. I was thinking `event_id` would be useful to identify events if there are multiple at the same timestamp. I don't think `host_name` and `pid` are necessary, but these fields were included for the non export events. We can remove these two for now so the schema is minimal